sdks/python/apache_beam/runners/dataflow/dataflow_metrics.py

```diff
@@ -210,6 +210,14 @@ def _get_metric_value(self, metric):
       dist_sum = _get_match(
           metric.distribution.object_value.properties,
           lambda x: x.key == 'sum').value.integer_value
+      if not dist_sum:
+        # distribution metric is not meant to use on large values, but in case
+        # it is, the value can overflow and become double_value, the correctness
+        # of the value may not be guaranteed.
+        dist_sum = int(
```

Comment:
Do we need to log if this happens else this will be a silent failure.

Reply:
Sounds good.

Reply:
Should we also add the metric name in the logs for easier debugging.

Reply:
done.